lambda-buffers-codegen/data/rust-plutus-pla.json

```diff
@@ -0,0 +1,158 @@
+{
+  "opaquesConfig": {
+    "Plutus.V1.PlutusData": [
+      "plutus-ledger-api",
+      "plutus_data",
+      "PlutusData"
+    ],
+    "Plutus.V1.Address": [
+      "plutus-ledger-api",
+      "v1::address",
+      "Address"
+    ],
+    "Plutus.V1.Credential": [
+      "plutus-ledger-api",
+      "v1::address",
+      "Credential"
+    ],
+    "Plutus.V1.StakingCredential": [
+      "plutus-ledger-api",
+      "v1::address",
+      "StakingCredential"
+    ],
+    "Plutus.V1.PubKeyHash": [
+      "plutus-ledger-api",
+      "v1::crypto",
+      "Ed25519PubKeyHash"
```

Comment:
I don't think that is correct.

https://github.com/input-output-hk/plutus/blob/9607a642496768f756d5bfb21c2a65c04788b576/plutus-ledger-api/src/PlutusLedgerApi/V1/Crypto.hs#L29

There's no mention of Ed25519, and it's just PubKeyHash.

If it is correct that this is infact the Ed25519 pub key, I still think that we should probably stick to the original naming.

Reply:
hmm, I understand this is a bit controversial, but I think being explicit with the cryptographic scheme is an improvement over the original API.
We worked with Schnorr and ecdsa keys in our last project, and it just made more sense to be precise.